lwotgtelem/gnmit/subscribe/subscribe.go

```diff
@@ -0,0 +1,596 @@
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package subscribe implements the gnmi.proto Subscribe service API.
+// This is a forked version of github.com/openconfig/gnmi's subscribe package.
```

Comment:
worth mentioning why the fork is needed (what functionality is added) here

Reply:
I added something here to state what we added in `gnmit` -- particularly its the ability to have tasks that create telemetry data, and have a local subscriber to the cache.